driver/bpf/fillers.h

```diff
@@ -1066,16 +1107,17 @@ FILLER(sys_getrlimit_setrlrimit_x, true)
 		max = -1;
 	}
 
-	/*
-	 * cur
-	 */
+	/* Parameter 2: cur (type: PT_ERRNO) */
 	res = bpf_push_s64_to_ring(data, cur);
 	CHECK_RES(res);
 
-	/*
-	 * max
-	 */
-	return bpf_push_s64_to_ring(data, max);
+	/* Parameter 3: max (type: PT_ERRNO) */
+	res = bpf_push_s64_to_ring(data, max);
+	CHECK_RES(res);
+	
+	/* Parameter 4: resource (type: PT_ERRNO) */
+	unsigned long resource = bpf_syscall_get_argument(data, 0);
+	return bpf_push_u8_to_ring(data, rlimit_resource_to_scap(resource));
```

Comment:
`rlimit_resource_to_scap` should take an `uint32_t` and not an `unsigned long`

Reply:
@Andreagit97 But `rlimit_resource_to_scap ` takes in unsigned long.
https://github.com/falcosecurity/libs/blob/4ed2a61da902a4ed37a5a932ca588ac30e0452d0/driver/ppm_flag_helpers.h#L1291-L1331

Reply:
Yeah we should change its signature into
```c
static __always_inline uint8_t rlimit_resource_to_scap(uint32_t resource) 
```